### PR TITLE
fix: updating MFJA README with new env variables

### PR DESCRIPTION
**Pull request by rutujasurve94**
Changing `DATA_PATH` to `JINA_DATA_PATH`, adding `jina_version` option for CLI on running `cookiecutter`.

### CODE REVIEW

**Comment by maximilianwerk** [comment]
When we change the `SHARDS` in the cookiecutter to `JINA_SHARDS` we also need to change it here in the `README.md`.

**Review by maximilianwerk — APPROVED** [review]
LGTM

File: my-first-jina-app/README.md [hunk 1]
```diff
@@ -470,7 +470,7 @@ Starts our Pods section, and specifies our first Pod, named `crafter` which is d
 ```yaml
   encoder:
     yaml_path: pods/encode.yml
-    replicas: $REPLICAS
+    parallel: $PARALLEL
```
Comment (maximilianwerk) [inline my-first-jina-app/README.md:473]: `JINA_PARALLEL`

File: my-first-jina-app/README.md [hunk 2]
```diff
@@ -493,7 +493,7 @@ This code specifies:
 ```yaml
   chunk_idx:
     yaml_path: pods/chunk.yml
-    replicas: $JINA_SHARDS
+    parallel: $JINA_SHARDS
```
Comment (maximilianwerk) [inline my-first-jina-app/README.md:496]: Either `shards: $JINA_SHARDS` or `parallel: $JINA_PARALLEL`

**Review by maximilianwerk — CHANGES_REQUESTED** [review]

File: my-first-jina-app/README.md (hunk 1 above)
Comment (rutujasurve94) [reply my-first-jina-app/README.md]: done

File: my-first-jina-app/README.md (hunk 2 above)
Comment (rutujasurve94) [reply my-first-jina-app/README.md:496]: done

**Review by maximilianwerk — APPROVED** [review]
LGTM

**Pull request merged**